lithops/standalone/proxy.py

```diff
@@ -81,16 +113,24 @@ def run():
     """
     global last_usage_time
     global jobs
+    global auto_dismantle
+    global soft_dismantle_timeout
+    global hard_dismantle_timeout
 
     message = flask.request.get_json(force=True, silent=True)
     if message and not isinstance(message, dict):
         return error()
 
     last_usage_time = time.time()
 
+    standalone_config = message['config']['standalone']
+    auto_dismantle = standalone_config['auto_dismantle']
+    soft_dismantle_timeout = standalone_config['soft_dismantle_timeout']
+    hard_dismantle_timeout = standalone_config['hard_dismantle_timeout']
+
     act_id = str(uuid.uuid4()).replace('-', '')[:12]
     runtime = message['job_description']['runtime_name']
-    logger.info("Running job in {}".format(runtime))
+    logger.info("Running job in {}. Check /tmp/lithops/local_handler.log for execution logs".format(runtime))
```

Comment:
currently the file is proxy.log

Reply:
The proxy is a frontend interface that receives jobs, put them into execution in an asynchronous way, and forgets about execution. The actual function logs are located in /tmp/lithops/local_handler.log as stated in the message